security.md

```diff
@@ -12,11 +12,23 @@ Security announcements related to Scala are published to the ["Security Announce
 Messages to this channel can only be posted by administrators, so it is very low traffic.
 To set up email notifications for new security announcements, read [this post](https://users.scala-lang.org/t/about-the-security-announcements-category).
 
+## Releases Integrity
+
+To ensure the integrity of all the releases, our organization uses [PGP](https://gnupg.org/) keys for cryptographic signing. 
+We provide below an **exhaustive** list of all the keys
+used for signing the artifacts under the `org.scala-lang` namespace on [Maven Central](https://central.sonatype.com/namespace/org.scala-lang)
+
+|                    Fingerprint                    | Algorithm |                                                       Public Key                                                        |
+|:-------------------------------------------------:|:---------:|:-----------------------------------------------------------------------------------------------------------------------:|
+| 86DA 41A5 E169 9C9C EBE9 64A8 A905 2B1B 6D92 E560 | RSA-4096  | [Download Public Key](https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x86da41a5e1699c9cebe964a8a9052b1b6d92e560) |
```

Comment:
As mentioned by @SethTisue - this key - which expired:
```
$ gpg --list-public-keys C03EF1D7D692BCFF
pub   rsa4096 2018-02-13 [SC] [expired: 2020-02-13]
      86DA41A5E1699C9CEBE964A8A9052B1B6D92E560
uid           [ expired] Scala Project <security@scala-lang.org>
sub   rsa4096 2018-02-13 [E] [expired: 2020-02-13]
sub   rsa4096 2018-02-14 [S] [expired: 2020-02-13]
```
was used to create signature:
```
$ gpg --verify org/scala-lang/scala-library/2.13.15/scala-library-2.13.15.pom.asc org/scala-lang/scala-library/2.13.15/scala-library-2.13.15.pom
gpg: Signature made Fri 20 Sep 2024 21:25:22 CEST
gpg:                using RSA key C03EF1D7D692BCFF
gpg: Good signature from "Scala Project <security@scala-lang.org>" [expired]
```
after expiration time.

So either the signing tool was forced to use expired key, or the key at signature creation place was updated and not expired at the time. In the latter case - may I ask for that key to be sent to keyservers, please?